test_runner/batch_others/test_tenant_conf.py

```diff
@@ -0,0 +1,32 @@
+from contextlib import closing
+
+import pytest
+
+from fixtures.zenith_fixtures import ZenithEnvBuilder
+
+
+def test_tenant_config(zenith_env_builder: ZenithEnvBuilder):
```

Comment:
I propose to test the config update here too.
Would be amazing to update it to something ridiculous and then try to run some operation to see if the config change had influenced that, but getting the value is also ok, I guess.